crossbeam/datasets/bottom_up_data_generation.py

```diff
@@ -202,7 +206,7 @@ def main(argv):
     pool = multiprocessing.Pool(FLAGS.num_datagen_proc)
     seeds = list(range(FLAGS.data_gen_seed, FLAGS.data_gen_seed + FLAGS.num_searches))
     total_num_tasks = 0
-    n_shards = 0
+    n_shards = FLAGS.data_gen_seed
```

Comment:
n_shards is the current shard index right? we do want shards to count from 0 upward. but we don't necessarily want seeds to count from 0 upward, since we need different seeds for different splits of the dataset (each split should have its own shard 0 but using different seeds).

so "seed" and "shard index" should be tracked separately